### PR TITLE
[zh-cn] Update access-cluster.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -335,23 +335,6 @@ Python 客户端可以像 kubectl CLI 一样使用相同的
 为其它语言提供访问 API 的方法。
 参阅其它库的相关文档以获取他们是如何验证的。
 
-<!--
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/access-cluster.md:338]: Line 338 - 349 shall not be deleted.

**Comment by Sea-n** [comment]
Hey @gaogao101,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by Sea-n** [comment]
Hey @gaogao101,

Thanks for proposing this PR. But since you didn't respond for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.
(Note that please don't push anything while PR closed, or you won't be able to reopen it.)

**Pull request closed without merging**